src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/EditContactDetails.cshtml

```diff
@@ -20,7 +20,14 @@
 }
 @section full_main_content
 {
-    @(this.WeeeGds().BackLink(@Url.UrlFor<ProducerController>(a => a.TaskList())))
+    @if (Model.RedirectToCheckAnswers == true)
```

Comment:
@will-hope98 if we extract this into an action into the controller called "BackToPrevious" or something and then redirect there, we wouldnt need this duplicated condition in the views

Reply:
Done